web/profiles/custom/yalesites_profile/modules/custom/ys_themes/js/levers.js

```diff
@@ -0,0 +1,52 @@
+((Drupal) => {
+  Drupal.behaviors.ysLevers = {
+    attach: function (context, settings) { // eslint-disable-line
+      once("ysLevers", ".layout-container", context).forEach((element) => {
```

Comment:
Just thinking aloud, the once...forEach pattern is so common in Drupal that I don't think we want to change this. But in reality, this is superfluous code. Even if there are multiple 'layout-containers' on a site, we don't want to change the :root element multiple times. Nothing to resolve unless you want to pull at that string.

Reply:
I get why Drupal does this - for all those pesky ajax calls. No changes made, but I hear ya and I'd love to explore other options another time.